src/sentry/tasks/post_process.py

```diff
@@ -1017,7 +1017,7 @@ def process_code_mappings(job: PostProcessJob) -> None:
         else:
             return
 
-        auto_source_code_config.delay(project.id, event_id=event.event_id)
+        auto_source_code_config.delay(project.id, event_id=event.event_id, group_id=group_id)
```

Comment:
Just make sure that the `**kwargs` in the definition is already deployed before this deploys. I assume it is, just want to be doubly sure